From: Ilias Tsitsimpis <i.tsitsimpis@gmail.com>
Date: Tue, 10 Nov 2015 17:33:52 +0200
Subject: Use imaplib2 package instead of embedded copy

According to Debian Policy 4.13, Debian packages should not make use of
convenience copies unless the included package is explicitly intended to
be used in this way. Since OfflineIMAP's imaplib2 doesn't diverge from
upstream, use the python-imaplib2 package instead of the embedded copy.

Bug-Debian: https://bugs.debian.org/728445
Forwarded: no (Debian specific)
---
 offlineimap/folder/IMAP.py | 2 +-
 offlineimap/imaplibutil.py | 2 +-
 offlineimap/init.py        | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/offlineimap/folder/IMAP.py b/offlineimap/folder/IMAP.py
index 60b5301..62e8ca0 100644
--- a/offlineimap/folder/IMAP.py
+++ b/offlineimap/folder/IMAP.py
@@ -25,7 +25,7 @@ from sys import exc_info
 from .Base import BaseFolder
 from offlineimap import imaputil, imaplibutil, emailutil, OfflineImapError
 from offlineimap import globals
-from offlineimap.imaplib2 import MonthNames
+from imaplib2 import MonthNames
 
 
 # Globals
diff --git a/offlineimap/imaplibutil.py b/offlineimap/imaplibutil.py
index 952404a..b102e63 100644
--- a/offlineimap/imaplibutil.py
+++ b/offlineimap/imaplibutil.py
@@ -26,7 +26,7 @@ import errno
 
 from offlineimap.ui import getglobalui
 from offlineimap import OfflineImapError
-from offlineimap.imaplib2 import IMAP4, IMAP4_SSL, zlib, InternalDate, Mon2num
+from imaplib2 import IMAP4, IMAP4_SSL, zlib, InternalDate, Mon2num
 
 
 class UsefulIMAPMixIn(object):
diff --git a/offlineimap/init.py b/offlineimap/init.py
index d94dcb7..bc13c5c 100644
--- a/offlineimap/init.py
+++ b/offlineimap/init.py
@@ -18,7 +18,7 @@
 import os
 import sys
 import threading
-import offlineimap.imaplib2 as imaplib
+import imaplib2 as imaplib
 import signal
 import socket
 import logging
